[Echo] Support more types of prompt echo

AuferGachet opened the pull request:
- When the prompt type is **list[int]** or **list[list[int]]**, it needs to support echo after decoding.

AuferGachet committed:
wenxin-tools-700 When the prompt type is list[int] or list[list[int]], it needs to support echoing after decoding.

AuferGachet committed:
wenxin-tools-700 When the prompt type is list[int] or list[list[int]], it needs to support echoing after decoding.

AuferGachet committed:
wenxin-tools-700 When the prompt type is list[int] or list[list[int]], it needs to support echoing after decoding.

AuferGachet committed:
wenxin-tools-700 When the prompt type is list[int] or list[list[int]], it needs to support echoing after decoding.

LiqinruiG commented on fastdeploy/entrypoints/openai/serving_completion.py:
```diff
@@ -115,7 +115,15 @@ async def create_completion(self, request: CompletionRequest):
+                request_prompts = [self.engine_client.data_processor.tokenizer.decode(item) for item in request.prompt]
+            else:
+                request_prompts = request_prompt_ids
+            request.prompt = request_prompts
```
不去直接改 request.prompt。  在echo = true 的时候，再去编码。

AuferGachet committed:
wenxin-tools-700 When the prompt type is list[int] or list[list[int]], it needs to support echoing after decoding.

AuferGachet committed:
Merge branch 'develop' into echo_patch

AuferGachet committed:
wenxin-tools-700 When the prompt type is list[int] or list[list[int]], it needs to support echoing after decoding.

AuferGachet committed:
wenxin-tools-700 When the prompt type is list[int] or list[list[int]], it needs to support echoing after decoding.

AuferGachet committed:
wenxin-tools-700 When the prompt type is list[int] or list[list[int]], it needs to support echoing after decoding.

AuferGachet committed:
Merge branch 'develop' into echo_patch

luukunn committed:
Merge branch 'develop' into echo_patch

AuferGachet committed:
wenxin-tools-700 When the prompt type is list[int] or list[list[int]], it needs to support echoing after decoding.

AuferGachet committed:
update

Merge branch 'echo_patch' of https://github.com/AuferGachet/FastDeploy into echo_patch
